README.md

```diff
@@ -1,126 +1,248 @@
 # Leap
+Leap is blockchain node software and supporting tools that implement the [Antelope](https://github.com/AntelopeIO) protocol.
 
-Leap is blockchain node software and supporting tools that implements the [Antelope](https://github.com/AntelopeIO) protocol.
+## Branches
+The `main` branch is the development branch, do not use it for production. Refer to the [release page](https://github.com/AntelopeIO/leap/releases) for current information on releases, pre-releases, and obsolete releases, as well as the corresponding tags for those releases.
 
-## Repo Organization
+## Binary Installation
```

Comment:
> The change from "Software Installation" to "Binary Installation" breaks an anchor URL used in the docs. However, I can see that "binary" is more explicit.

Thank you. Yes, there are three ways to install the software so I wanted to make it explicit what we were doing. I thought about moving all the installation methods to the bottom in a single section, where "Software Installation" would be a good title, but I think we want the binary install up top so newcomers don't do an necessary build only to discover they can use binaries.

What would it take to change those broken links? Is that something I can do?

> Why not sticking to what we used before successfully: "Install from Binaries" for package installation[...]

I don't know, to be honest I don't remember what we had before. I just chose this because it is one word shorter. I am not married to it.

> [...]and "Build from Source" below for building from source, instead of "Source Installation" which is more rare in the literature since we don't really "install" from source.

I figure the end goal for a customer is to install what they built, unless building from source is an academic exercise. [The build](https://github.com/AntelopeIO/leap/blob/8c922dc6909a598020ea8106984c026c42898467/README.md#build) itself has its own section in the document. Moreover, "Source Installation" matches the pattern of "Binary Installation," so it makes it clear there are two options.

Since this section includes more than just a build, I would say "Build and Install from Source" or "Source Build and Install" would be the closest names to what you suggest that make sense for this heading without restructuring the document, which I am not opposed to.

Reply:
Ok we could use "Binary Installation" here and "Source Build and Install" below (although the latter sounds weird to me). We could also use "Install from Binaries" here and "Build and Install from Source" below. Just throw a dice and pick one 😄.

Reply:
Done.